python/cudf/cudf/core/series.py

```diff
@@ -984,6 +985,10 @@ def __repr__(self):
             for idx, value in enumerate(preprocess):
                 if value is None:
                     lines[idx] = lines[idx].replace(" NaT", "null")
+        if preprocess.dtype == np.dtype("object"):
+            for idx, value in enumerate(preprocess):
+                if value is None:
+                    lines[idx] = lines[idx].replace("<NA>", "null")
```

Comment:
Instead of doing this crazy preprocessing, can we just utilize `to_string(na_rep="null")`?